test/tags.js

```diff
@@ -11,7 +11,11 @@ var liquid = Liquid({
     ctx, src, dst;
 
 function test(src, dst) {
-    expect(liquid.parseAndRender(src, ctx)).to.equal(dst);
+    liquid.parseAndRender(src, ctx)
```

Comment:
Will do.  I haven't done any testing at all outside of one test at a time, and most of those are me actually debugging.  That's why I haven't looked into that yet.

I'm making good progress.  I pushed a commit to the asyncSpike branch of my fork just this morning that I'm hoping gets me past my biggest hurdle so far.  I'm ready to start testing everything again by removing the .only and getting the rest of the tags fixed up.  I expect to finish this weekend.